### DISCUSSION TITLE
Is there a reason some classes, like btn, have multiple values for the same property?

### DISCUSSION THREAD

**Discussion by ezeamin** [saadeghi/daisyui#2308] [Q&A] [answered]
The `.btn` class defines the following:

![image](https://github.com/saadeghi/daisyui/assets/82628932/5421b6de-e2a8-4fb1-a1b7-790fbeb126cf)

Is that ok?

**Comment by saadeghi** [comment, marked as answer]
Some of them exist because of merging the base skeleton styles (unstyled components) and the styled components (with design decisions) but you're right. They need some cleanups because this shouldn't go to production. I will fix this soon.
